charts/monitoring/node-exporter/values.yaml

```diff
@@ -12,33 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-nodeExporter:
-  image:
-    repository: quay.io/prometheus/node-exporter
-    tag: v1.7.0
-  # list of image pull secret references, e.g.
-  # imagePullSecrets:
-  #   - name: quay-io-pull-secret
+node-exporter:
+  # Check master values.yaml which are the default values for Chart version 4.44.0
+  # https://github.com/prometheus-community/helm-charts/blob/prometheus-node-exporter-4.44.0/charts/prometheus-node-exporter/values.yaml
   imagePullSecrets: []
-  resources:
-    requests:
-      cpu: 50m
-      memory: 24Mi
-    limits:
-      cpu: 250m
-      memory: 48Mi
```

Comment:
Requests/limits should not be removed and we should still keep some defaults here.